src/utils/currencies.ts

```diff
@@ -0,0 +1,293 @@
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Norwegian Krone NOK",
+        value: "NOK",
+        hasSymbol: "kr",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Sri Lankan Rupee LKR",
+        value: "LKR",
+        hasSymbol: "Rs",
+        maxFractionalDigits: 2
+    },
+    { label: "Swiss Franc CHF", value: "CHF", maxFractionalDigits: 2 },
+    {
+        label: "United Arab Emirates Dirham AED",
+        value: "AED",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "United States Dollar USD",
+        value: "USD",
+        hasSymbol: "$",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Malaysian Ringgit MYR",
+        value: "MYR",
+        hasSymbol: "RM",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Nigerian Naira NGN",
+        value: "NGN",
+        hasSymbol: "₦",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "New Zealand Dollar NZD",
+        value: "NZD",
+        hasSymbol: "$",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Philippine Peso PHP",
+        value: "PHP",
+        hasSymbol: "₱",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Pakistani Rupee PKR",
+        value: "PKR",
+        hasSymbol: "₨",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Polish Złoty PLN",
+        value: "PLN",
+        hasSymbol: "zł",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Russian Ruble RUB",
+        value: "RUB",
+        hasSymbol: "₽",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Saudi Riyal SAR",
+        value: "SAR",
+        hasSymbol: "SR",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Swedish Krona SEK",
+        value: "SEK",
+        hasSymbol: "kr",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Singapore Dollar SGD",
+        value: "SGD",
+        hasSymbol: "$",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Thai Baht THB",
+        value: "THB",
+        hasSymbol: "฿",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Turkish Lira TRY",
+        value: "TRY",
+        hasSymbol: "₺",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "New Taiwan Dollar TWD",
+        value: "TWD",
+        hasSymbol: "NT$",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Ukrainian Hryvnia UAH",
+        value: "UAH",
+        hasSymbol: "₴",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Venezuelan Bolívar VEF",
+        value: "VEF",
+        hasSymbol: "Bs",
+        maxFractionalDigits: 2
+    },
+    {
+        label: "Vietnamese Dong VND",
+        value: "VND",
+        hasSymbol: "₫",
+        maxFractionalDigits: 0
+    },
+    {
+        label: "South African Rand ZAR",
+        value: "ZAR",
+        hasSymbol: "R",
+        maxFractionalDigits: 2
+    }
+].sort((a, b) => (a.value > b.value ? 1 : b.value > a.value ? -1 : 0));
```

Comment:
this list looks already alphabetical, I think we can drop the sort

Reply:
What do you think makes more sense, value sorted or label sorted?

Reply:
went with value sorted for now